### PR TITLE
Mixpanel integration

### PR DESCRIPTION
**Pull request by deanrobertcook**
@shumengye FYI

#### APK
[Download build #9513](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9513/artifact/build/artifact/wire-dev-PR1142-9513.apk)
[Download build #9514](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9514/artifact/build/artifact/wire-dev-PR1142-9514.apk)
[Download build #9517](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9517/artifact/build/artifact/wire-dev-PR1142-9517.apk)
[Download build #9537](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9537/artifact/build/artifact/wire-dev-PR1142-9537.apk)
[Download build #9538](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9538/artifact/build/artifact/wire-dev-PR1142-9538.apk)
[Download build #9539](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9539/artifact/build/artifact/wire-dev-PR1142-9539.apk)
[Download build #9546](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9546/artifact/build/artifact/wire-dev-PR1142-9546.apk)
[Download build #9547](http://192.168.10.18:8080/job/Pull%20Request%20Builder/9547/artifact/build/artifact/wire-dev-PR1142-9547.apk)

### CODE REVIEW

**Comment by deanrobertcook** [comment]
build again

**Comment by someoneEsle** [comment]
I'm not sure what you want to do with this, but: 1:1 team conversations are marked as "group" conversations in the log.

**Comment by deanrobertcook** [comment]
build again

File: app/src/main/java/com/waz/zclient/utils/HockeyCrashReporting.java [hunk 1]
```diff
@@ -54,10 +52,10 @@ public String getUserID() {
         CrashManager.initialize(context, Util.getAppIdentifier(context), listener);
 
         boolean nativeCrashFound = NativeCrashManager.loggedDumpFiles(Util.getAppIdentifier(context));
-        if (nativeCrashFound) {
-            StringBuilder details = new StringBuilder(Constants.PHONE_MANUFACTURER).append("/").append(Constants.PHONE_MODEL);
-            trackingController.tagEvent(ExceptionEvent.exception("NDK", details.toString()));
-        }
+//        if (nativeCrashFound) {
```
Comment (JPonte) [inline app/src/main/java/com/waz/zclient/utils/HockeyCrashReporting.java]: Any reason to leave this commented out?

Comment (deanrobertcook) [reply app/src/main/java/com/waz/zclient/utils/HockeyCrashReporting.java]: Will fix in another PR

**Pull request merged**